bleak/exc.py

```diff
@@ -8,6 +9,26 @@ class BleakError(Exception):
     pass
 
 
+class BleakCharacteristicNotFoundError(BleakError):
+    """
+    Exception which is raised if a device does not support a characteristic.
+
+    .. versionadded: 0.22.0
+    """
+
+    char_specifier: Union[int, str, uuid.UUID]
+
+    def __init__(
+        self, char_specifier: Union[int, str, uuid.UUID], *args: object
```

Comment:
What is `*args` for? Can we leave it out?

Reply:
It's there just to stick to the same signature as that of `Exception.__init__`, but I think it can be omitted.
@bdraco do you agree?

Reply:
Seems fine to omit 👍